app/views/catarse_bootstrap/explore/index.html.slim

```diff
@@ -35,15 +35,15 @@
             .w-col.w-col-2.w-col-small-6.w-col-tiny-6
               a.explore-link.w-inline-block.btn-category [
                 class="#{'double-line' if category.to_s.size > 13}"
-                href="#by_category_id/#{category.id}" 
-                data-name=(params[:locale] == 'en' ? category.name_en : category.name_pt)
+                href="#by_category_id/#{category.id}"
+                data-name=( (category.name_fr if params[:locale] == 'fr') || (category.name_en if params[:locale] == 'en') || category.name_pt )
```

Comment:
Let's make this more generic and move the logic to a category decorator. Something like:
def display_name
I18n.available_locales.include?(params[:locale].to_sym) ? source.send('name_'+params[:locale]) : source.name_pt
end
Not actually tested this but you get the idea...